### PR TITLE
List BitDaric (DARX)

### PR DESCRIPTION
**Pull request by bitdaric**
- Official project URL: http://www.bitdaric.live
- Official block explorer URL: https://explorer.bitdaric.live

### CODE REVIEW

**Review by blabno — CHANGES_REQUESTED** [review]
NACK apply Bisq code style

**Comment by ManfredKarrer** [comment]
Close because of inactivity.

**Comment by bitdaric** [comment]
sorry but I didn`t get it. what should I do??

**Comment by blabno** [comment]
Apply Bisq code style

File: assets/src/main/java/bisq/asset/coins/BitDaric.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ */
+package bisq.asset.coins;
+import bisq.asset.AddressValidationResult;
+import bisq.asset.AddressValidator;
+import bisq.asset.Coin;
+public class BitDaric extends Coin {
+    public BitDaric() {
+        super("BitDaric", "DARX", new BitDaricAddressValidator());
+    }
+    public static class BitDaricAddressValidator implements AddressValidator {
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/BitDaric.java:25]: Do not introduce new validator. Use RegexAddressValidator.

**Review by blabno — COMMENTED** [review]
NACK per comments

File: assets/src/main/java/bisq/asset/coins/BitDaric.java (hunk 1 above)
Comment (bitdaric) [reply assets/src/main/java/bisq/asset/coins/BitDaric.java:25]: done https://github.com/bisq-network/bisq/pull/2234

**Pull request closed without merging**